core/shared/src/test/scala/io/github/timwspence/cats/stm/STMSpec.scala

```diff
@@ -29,7 +29,7 @@ import scala.util.Random
   */
 class STMSpec extends CatsEffectSuite {
 
-  val stm = STM.runtime[IO].unsafeRunSync()
```

Comment:
Was this even safe before? I think I used to do this a bunch in tests and I've recently been wondering if it was ever even safe to do so